docs/getting-started.md

```diff
@@ -34,221 +37,379 @@ The high level data flow is as follows:
 
-## 5 Query your local Graph Node
-With your subgraph deployed to your locally running Graph Node, visit http://127.0.0.1:8000/ to open up a [Graphiql](https://github.com/graphql/graphiql) interface where you can explore your deployed GraphQL API for your subgraph by issuing queries and viewing the schema.
+Typically ganache runs on port `9545`, so you would run the same command for a local geth or parity node, but replace `8545` with `9545`. 
 
-See the [Query API](graphql-api.md#1-queries) for a complete reference on how to query your subgraph's entities.
+##### 4.2.3.3 Local Parity dev Chain (Works Similar to Ganache) 
+
+To setup a local testnet that will allow you to rapidly test the project, download the parity software if you don’t already have it. This command will work for a one line install:
+
+`bash <(curl https://get.parity.io -L)`
+
+Next you want to make an account that you can unlock, and make transactions on for the parity dev chain. run the following command:
+
+`parity account new --chain dev`
+
+Make a password you will remember. Make note of the account that gets output. Now you also have to make that password a text file, and pass it into the next command. The desktop is a good location for it. If the password is `123`, only put the numbers in the text file. Do not include any quotations. Then you can run this command:
+
+`parity --config dev --unsafe-expose --jsonrpc-cors="all" --unlock <ETHEREUM_ADDRESS> --password ~/Desktop/password.txt`
+
+The chain should start! It is a chain with 0 block time and instant transactions, making testing very fast. Passing `unsafe-expose` and ` --jsonrpc-cors="all"` as flags allow Metamask to connect. 
+
+This chain will be accessible on `localhost:8545`. All extra information for customization of a parity dev chains is located [here](https://wiki.parity.io/Private-development-chain#customizing-the-development-chain).
+
+Next you need to import the account with the private testnet ether to Metamask. The base account that the normal configuration of parity gives you is `0x00a329c0648769A73afAc7F9381E08FB43dBEA72`. Its private key is:
+
+Private key : 4d5db4107d237df6a3d58ee5f70ae63d73d7658d4026f2eefd2f204c81682cb7 (note this is the private key given along with the parity dev chain, so it is okay to share)
+
+Use Metamask --> import account ---> private key.
+
+You now have an Ethereum account with a ton of ether, and you should be able to set up the migrations on this network and use Truffle. Now, send some ether to the previous account that got created and unlocked, this way you can run `truffle migrations` with this account. Weirdly,  `0x00a329c0648769A73afAc7F9381E08FB43dBEA72` does not get unlocked by parity, probably because it's password is an empty string, and it might not read properly from a blank text file. Anyways, that's why we have to make a new account, and send the test either from the default account. 
```

Comment:
`ether` -> `Ether`?
`truffle migrations` -> `truffle migrate`
`it's password` -> `its password`
`either` -> `Ether`

Reply:
`from the default account` -> `from the default account to this new account`